api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerConsumerSpec.java

```diff
@@ -58,4 +60,14 @@ public String getGroupId() {
     public void setGroupId(String groupId) {
         this.groupId = groupId;
     }
+
+    @Description("Offset commit interval in ms. Default value is 60 000.")
```

Comment:
why a space in "60 000"?

Reply:
It is fixed in https://github.com/strimzi/strimzi-kafka-operator/pull/1768/files#diff-d79dc54e295cffd73c71facebff75566R64

Reply:
I don't know why github showed me the old version! :-)

Reply:
It is another PR. I had to update the `spec` file to apply suggested changes in the docs.